spec/classes/node_spec.rb

```diff
@@ -158,6 +175,9 @@
         it do
           is_expected.to contain_file('/etc/xylem/xylem.yml')
             .with_content(match_yaml({
+                'backend' => 'rhumba.backends.redis',
+                'redis_host' => '127.0.0.1',
+                'redis_port' => 6379,
```

Comment:
We should probably have a set of default config entries: `@default_config = {'backend' => 'rhumba.backends.redis', ...}`
Then we use `.with_content(match_yaml(@default_config.merge {'queues' => ...})` to avoid all the duplication and reduce churn when the defaults change.